Shared/UI/Views/MessageContentView/MessageContentView.swift

```diff
@@ -73,15 +73,12 @@ class MessageContentView: View {
             self.authorView.isVisible = state == .expanded
             self.layoutNow()
         }.store(in: &self.cancellables)
-    }
 
-    #if IOS
-    func setContextMenu() {
-        self.bubbleView.interactions.removeAll()
+#if IOS
         let contextMenuInteraction = UIContextMenuInteraction(delegate: self)
         self.bubbleView.addInteraction(contextMenuInteraction)
+#endif
     }
-    #endif
```

Comment:
Enabling context menu interactions needs to be changed frequently because a cell's "frontmost" status can change. To that end, I'm adding the context menu interaction just once and changing the cell's user interaction flag dynamically.